src/aura/autocomplete/autocomplete.cmp

```diff
@@ -3,28 +3,31 @@
 
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
     <aura:handler name="optionSelected" event="c:autocompleteSelectListOption" action="{!c.handleOptionSelected}"/>
+    <aura:handler name="change" value="{!v.isListVisible}" action="{!c.handleIsListVisibleChange}"/>
```

Comment:
Instead of using a change handler to react to changes of `isListVisible`, how about just using the attribute directly in the class string of the menu div?  `<div class="{!'slds-lookup__menu' + (isListVisible ? '' : ' slds-hide')}" ... >`

This way no change handler is needed.  `helper.setListVisibility` could be changed to just set the attribute, instead of manually toggling the css class.

Are there downsides to doing it like that?  It seems better than adding a change handler, to me, but what do you think?  I'm not sure if Aura will rerender the whole menu div if the class attribute needs to be updated, or if it will just update the attribute in the DOM...